CHANGELOG.md

```diff
@@ -2,31 +2,80 @@
 
 ## 0.8.0 (unreleased)
 
-The package name has changed from "crustls" to "rustls-ffi". If you are
-importing it as a library from other Rust code, you should import `rustls_ffi`.
-
-The header file is now named `rustls.h` instead of `crustls.h`, and the
-generated `.a` file is named `librustls_ffi.a` instead of `libcrustls.a`.
+The package name has changed to "rustls-ffi" (from "crustls").
+The header file (as installed by `make DESTDIR=/path/ install`)
+is now `rustls.h` and the library is `librustls.a`. The old library and header
```

Comment:
Ah, I must have gotten this mixed up with the `.a` file that gets installed to target/releases.